deploy/action.yml

```
@@ -2,27 +2,27 @@ name: Deploy Storefront
 description: Builds Middleware app docker image
```

```suggestion
description: Deploy the Storefront to the Console
```

If you could make this straight 🙏